### PR TITLE
Support integer id.

### PR DESCRIPTION
**Pull request by CPWstatic**
After https://github.com/vesoft-inc/nebula-common/pull/202

### CODE REVIEW

File: src/meta/processors/partsMan/CreateSpaceProcessor.cpp [hunk 1]
```diff
@@ -71,16 +72,30 @@ void CreateSpaceProcessor::process(const cpp2::CreateSpaceReq& req) {
...
+        return;
+    }
+    if (vidType != cpp2::PropertyType::INT64 && vidType != cpp2::PropertyType::STRING) {
+        LOG(ERROR) << "Create Space Failed : vid_type is illegal: "
+                   << meta::cpp2::_PropertyType_VALUES_TO_NAMES.at(vidType);
+        resp_.set_code(cpp2::ErrorCode::E_INVALID_PARM);
+        onFinished();
+        return;
+    }
+    if (vidType != cpp2::PropertyType::INT64 && vidSize != 8) {
```
Comment (panda-sheep) [inline src/meta/processors/partsMan/CreateSpaceProcessor.cpp:96]: What do you mean?

File: src/storage/exec/EdgeNode.h [hunk 2]
```diff
@@ -126,10 +126,10 @@ class FetchEdgeNode final : public EdgeNode<cpp2::EdgeKey> {
         }
         prefix_ = NebulaKeyUtils::edgePrefix(planContext_->vIdLen_,
                                              partId,
-                                             edgeKey.src,
+                                             edgeKey.src.getStr(),
                                              edgeKey.edge_type,
                                              edgeKey.ranking,
-                                             edgeKey.dst);
```
Comment (panda-sheep) [inline src/storage/exec/EdgeNode.h:132]: Have a doubt.
You have changed a lot of member functions of `NebulaKeyUtils`,
but have not seen the modification of the definition of these member functions?

File: src/meta/processors/partsMan/CreateSpaceProcessor.cpp (hunk 1 above)
Comment (CPWstatic) [reply src/meta/processors/partsMan/CreateSpaceProcessor.cpp:96]: Sorry, it was a mistabke.

File: src/storage/exec/EdgeNode.h (hunk 2 above)
Comment (CPWstatic) [reply src/storage/exec/EdgeNode.h:132]: Emm, I didn't change the utils itself.

**Review by dutor — DISMISSED** [review]
LGTM.

File: src/meta/processors/partsMan/CreateSpaceProcessor.cpp [hunk 3]
```diff
@@ -71,16 +72,30 @@ void CreateSpaceProcessor::process(const cpp2::CreateSpaceReq& req) {
...
-              return;
+            LOG(ERROR) << "Create Space Failed : replicaFactor is illegal: " << replicaFactor;
+            resp_.set_code(cpp2::ErrorCode::E_INVALID_PARM);
+            onFinished();
+            return;
         }
         // Set the default value back to the struct, which will be written to storage
         properties.set_replica_factor(replicaFactor);
     }
     if (vidSize <= 0 && vidSize > std::numeric_limits<int32_t>::max()) {
```
Comment (laura-ding) [inline src/meta/processors/partsMan/CreateSpaceProcessor.cpp:83]: graphd limit it less than std::numeric_limits<int16_t>::max(). here need to unify

**Review by laura-ding — APPROVED** [review]

**Review by laura-ding — APPROVED** [review]

**Review by dutor — APPROVED** [review]

**Pull request merged**